lib/remap-lang/src/parser.rs

```diff
@@ -381,18 +398,18 @@ mod tests {
             ),
             (
                 ".foo = to_string",
-                vec![" 1:8\n", "= expected if_statement, not, path, or block"],
+                vec![" 1:8\n", "= expected assignment, if_statement, not, or block"],
```

Comment:
More readable error reporting is something I'll be working on separately, so you can ignore the changes here and below.